accessapproval/apiv1/gapic_metadata.json

```diff
@@ -1,47 +1,47 @@
 {
-  "schema":  "1.0",
-  "comment":  "This file maps proto services/RPCs to the corresponding library clients/methods.",
-  "language":  "go",
-  "protoPackage":  "google.cloud.accessapproval.v1",
-  "libraryPackage":  "cloud.google.com/go/accessapproval/apiv1",
-  "services":  {
-    "AccessApproval":  {
-      "clients":  {
-        "grpc":  {
-          "libraryClient":  "Client",
-          "rpcs":  {
-            "ApproveApprovalRequest":  {
-              "methods":  [
+  "schema": "1.0",
+  "comment": "This file maps proto services/RPCs to the corresponding library clients/methods.",
+  "language": "go",
+  "protoPackage": "google.cloud.accessapproval.v1",
+  "libraryPackage": "cloud.google.com/go/accessapproval/apiv1",
+  "services": {
+    "AccessApproval": {
+      "clients": {
+        "grpc": {
+          "libraryClient": "Client",
```

Comment:
@noahdietz I think there is something up with the generator for this file. They diff on whitespace it looks like. Previous regens also sometimes included this but I had thought it was a fluke.

Reply:
I've seen this before...I will take a look, not really sure why it does this, it uses google.golang.org/protobuf/encoding/protojson to JSONify it.

Reply:
https://github.com/googleapis/gapic-generator-go/issues/691

Reply:
updated the issue.